confignode/src/main/java/org/apache/iotdb/confignode/manager/ConfigManager.java

```diff
@@ -47,6 +47,7 @@
 
   private LoadBalancer loadBalancer;
 
+  /** Test only constructor */
   public ConfigManager(String hashExecutorClass, int deviceGroupCount) {
```

Comment:
if a method only for test,  need add @TestOnly

Reply:
I found that @TestOnly is in org.apache.iotdb.db.utils.TestOnly package. But I couldn't dependence iotdb-server.

Reply:
Fix it